webpack.config.js

```diff
@@ -380,7 +380,7 @@ export default {
     entry: info.entries,
     // cockpit.js gets included via <script>, everything else should be bundled
     externals: (section === 'kdump/' || section === 'base1/') ? {} : { cockpit: "cockpit" },
-    plugins: plugins,
+    plugins,
```

Comment:
Ah, this slipped through -- vim ALE automatically fixes eslint issues. Hopefully acceptable.